src/components/icon/icon.spec.js

```diff
@@ -529,13 +529,18 @@ describe('mdIcon service', function() {
       });
     });
 
+    /*
+     * Previous to Angular 1.6, requesting an icon that is not found would throw no errors. After
+     * 1.6, since we do not have a .catch() handler, it now throws a Possibly Unhandled Rejection
+     * error.
+     */
     describe('icon is not found', function() {
       it('should not throw Error', function() {
         expect(function(){
           $mdIcon('notfound');
 
           $httpBackend.flush();
-        }).not.toThrow();
+        }).not.toThrow('Cannot GET notfoundicon.svg');
```

Comment:
I mean if you are verifying that `.not.toThrow('Cannot GET notfoundicon.svg');` the test would pass even if it throws `'Can't GET notfoundicon.svg'` (just because the error messahe is slightly different.

I am confused because I am not sure if you want to verify that it doesn't throw an error or that the promise isn't rejected. The way I'm thinking it, I would rather chain a `.catch()` callback and verify that it is called with the specified "thing" (object, thing, error) or not called (if that is what is supposed to happen).